### PR TITLE
weETH risk params update - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    IAaveV3ConfigEngine.BorrowUpdate[]
+      memory borrowUpdates = new IAaveV3ConfigEngine.BorrowUpdate[](1);
+
+    borrowUpdates[0] = IAaveV3ConfigEngine.BorrowUpdate({
+      asset: AaveV3ArbitrumAssets.weETH_UNDERLYING,
+      enabledToBorrow: EngineFlags.KEEP_CURRENT,
+      flashloanable: EngineFlags.KEEP_CURRENT,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol:58]:
```suggestion
      stableRateModeEnabled: EngineFlags.KEEP_CURRENT,
      borrowableInIsolation: EngineFlags.KEEP_CURRENT,
      withSiloedBorrowing: EngineFlags.KEEP_CURRENT,
```

Should be `EngineFlags.KEEP_CURRENT` as we are not changing these params

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  {
+    IAaveV3ConfigEngine.RateStrategyUpdate[]
+      memory rateStrategies = new IAaveV3ConfigEngine.RateStrategyUpdate[](1);
+    rateStrategies[0] = IAaveV3ConfigEngine.RateStrategyUpdate({
+      asset: AaveV3ArbitrumAssets.weETH_UNDERLYING,
+      params: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(35_00),
+        baseVariableBorrowRate: _bpsToRay(0),
+        variableRateSlope1: _bpsToRay(7_00),
+        variableRateSlope2: _bpsToRay(300_00),
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol:32]:
```suggestion
        baseVariableBorrowRate: EngineFlags.KEEP_CURRENT,
        variableRateSlope1: EngineFlags.KEEP_CURRENT,
        variableRateSlope2: EngineFlags.KEEP_CURRENT,
```
Variable interest rates is not being changed so I would use `EngineFlags.KEEP_CURRENT`

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(300_00),
+        baseStableRateOffset: _bpsToRay(0),
+        stableRateExcessOffset: _bpsToRay(0),
+        optimalStableToTotalDebtRatio: _bpsToRay(0)
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol:37]: What is the reason for changing the stable interest rates? 
It should not make any difference no as stable borrowing is anyway disabled and being deprecated. 

If the reason is to align stable rates across all network for weETH, even doing so there will be inconsistency as stable rate slope 1 and stable rate slope 2 for weETH on mainnet is 0%.

Reference: [Dashboard Link](https://dash.onaave.com/#chainId=&type=&symbol=weETH&fields=symbol%2CbaseStableBorrowRate%2CbaseVariableBorrowRate%2CstableRateSlope1%2CstableRateSlope2%2CvariableRateSlope1%2CvariableRateSlope2%2CstableBorrowRateEnabled%2CborrowableInIsolation%2CisSiloedBorrowing)

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+| Variable Base            | 0.0%           |
+| Variable Slope1          | 7.00%          |
+| Variable Slope2          | 300.00%        |
+| Uoptimal                 | 35.00%         |
+| Reserve Factor           | 45.00%         |
+| Stable Borrowing         | Disabled       |
+| Flashloanable            | Yes            |
+| Siloed Borrowing         | No             |
+| Borrowed in Isolation    | No             |
+| E-Mode Category          | ETH-correlated |
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md:51]: I would only include risk params in the table which are being changed.

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+The current proposal suggests adjusting the Interest Rate Curve for weETH on Arbitrum network to align with the mainnet, so Risk Parameters will be the same for weETH on Mainnet, Arbitrum, and Base networks.
+
+The aim is to optimize the utilization rates and improve both lending and borrowing for users.
+
+## Motivation
+
+By adjusting the Interest Rate Curve, we aim to encourage more borrowing and lending activity, optimize the utilization rates, and enhance liquidity on Arbitrum and Base networks.
+
+1. **Base Rate Increase:** Increasing the base rate provides a more attractive return for lenders, encouraging more weETH deposits.
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md:18]: The variable base rate on the payload is the same and not being changed

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+2. **Slope 1 Adjustment:** Adjusting the Stable Rate Slope offers slightly higher interest for moderate utilization, making borrowing more attractive.
+3. **Slope 2 Adjustment:** Reducing Stable Rate Slope 2 prevents excessively high borrowing costs at higher utilization rates, thus avoiding sharp declines in borrowing activities.
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md:20]: @marczeller Why are we changing the stable rate slope 1 and 2? It should not matter no as stable rates are disabled or am I missing something?

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol (hunk 3 above)
Comment (marczeller) [reply src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol:37]: you're correct we do not want to change the stable rate. KEEP_CURRENT was initial intent

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md (hunk 6 above)
Comment (marczeller) [reply src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AdjustingInterestRateCurveForWeETHOnArbitrum.md]: fixed

File: src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+      params: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(35_00),
+        baseVariableBorrowRate: EngineFlags.KEEP_CURRENT,
+        variableRateSlope1: EngineFlags.KEEP_CURRENT,
+        variableRateSlope2: EngineFlags.KEEP_CURRENT,
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(300_00),
+        baseStableRateOffset: _bpsToRay(0),
+        stableRateExcessOffset: _bpsToRay(0),
+        optimalStableToTotalDebtRatio: _bpsToRay(0)
```
Comment (brotherlymite) [inline src/20240603_AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum/AaveV3Arbitrum_AdjustingInterestRateCurveForWeETHOnArbitrum_20240603.sol:37]:
```suggestion
        stableRateSlope1: EngineFlags.KEEP_CURRENT,
        stableRateSlope2: EngineFlags.KEEP_CURRENT,
        baseStableRateOffset: EngineFlags.KEEP_CURRENT,
        stableRateExcessOffset: EngineFlags.KEEP_CURRENT,
        optimalStableToTotalDebtRatio: EngineFlags.KEEP_CURRENT
```
Also please update the generated diff

**Review by ianflexa — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**